src/components/application_manager/src/commands/mobile/on_system_request_notification.cc

```diff
@@ -77,25 +77,22 @@ void OnSystemRequestNotification::Run() {
     return;
   }
 
-  if (RequestType::PROPRIETARY == request_type) {
-    /* According to requirements:
-       "If the requestType = PROPRIETARY, add to mobile API fileType = JSON
-        If the requestType = HTTP, add to mobile API fileType = BINARY"
-       Also in Genivi SDL we don't save the PT to file - we put it directly in
-       binary_data */
-
-    const std::string filename =
-        (*message_)[strings::msg_params][strings::file_name].asString();
-    BinaryMessage binary_data;
-    file_system::ReadBinaryFile(filename, binary_data);
+  const std::string filename =
```

Comment:
@LitvinenkoIra Why not reference?

Reply:
@AByzhynar Fixed 830b1677e24e1b14cd2c15b8c02f71186eedd054